settings/src/main/java/fr/pilato/elasticsearch/crawler/fs/settings/Rest.java

```diff
@@ -0,0 +1,85 @@
+package fr.pilato.elasticsearch.crawler.fs.settings;
+
+import java.util.Objects;
+
+public class Rest {
```

Comment:
This change makes sense to me.
Could you change the javadoc in `ServerUrl` class?
I wrote:

```
/**
 * This class represents a ServerUrl which is basically just a String which
 * can be either a url or a cloud id.
 * This is used in Rest and Elasticsearch.Node classes.
 */
```

The last line is not true anymore and only applies to `Elasticsearch.Node` class.